services/user.py

```diff
@@ -0,0 +1,53 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
+
+def create_user(username: str,
+                password: str,
+                email: str = None,
+                first_name: str = None,
+                last_name: str = None) -> User:
+    user = get_user_model().objects.create_user(username=username,
+                                                password=password
+                                                )
```

Comment:
If u change optional default parametrs to  `""` u can write 
```return get_user_model().objects.create_user(
        username=username,
        password=password,
        email=email,
        first_name=first_name,
        last_name=last_name
    )```. Basically if u don't put smth in email etc django fill with all values with `""`. I am not sure mb its bad practise

Reply:
it is better to use None as defaul value here